usagov/usagov-2021
# USAGOV-1957: jq errors during www bootstrap

https://cm-jira.usa.gov/browse/USAGOV-1957

## Description
Updated `bootstrap.sh` to notice when the 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
On our locals, you can edit `docker-compose.yml` and remove the `aws-rds` array _(remove lines 51 through 60)_ and then run `bin/bootstrap` - about three lines or so after the `Deployment: bootstrap starting` message you should see a message that says `WARNING: The aws-rds variable is not set [...]` and *not* see a message like `jq: error (at <stdin>:1): Cannot iterate over null (null)`

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
